packages/zoltan2/src/algorithms/color/Zoltan2_AlgHybridGMB.hpp

```diff
@@ -0,0 +1,458 @@
+#ifndef _ZOLTAN2_ALGHYBRIDGMB_HPP_
+#define _ZOLTAN2_ALGHYBRIDGMB_HPP_
+
+#include <vector>
+#include <unordered_map>
+#include <iostream>
+#include <queue>
+#include <unistd.h>
+
+#include <Zoltan2_Algorithm.hpp>
+#include <Zoltan2_GraphModel.hpp>
+#include <Zoltan2_ColoringSolution.hpp>
+#include <Zoltan2_Util.hpp>
+#include <Zoltan2_TPLTraits.hpp>
+
+#include "Tpetra_Core.hpp"
+#include "Teuchos_RCP.hpp"
+#include "Tpetra_Import.hpp"
+#include "Tpetra_FEMultiVector.hpp"
+
+#include <KokkosKernels_Handle.hpp>
+#include "KokkosKernels_IOUtils.hpp"
+#include "KokkosGraph_Distance1Color.hpp"
+#include "KokkosGraph_Distance1ColorHandle.hpp"
+
+#include "zz_rand.h"
+//////////////////////////////////////////////
+//! \file Zoltan2_AlgHybridGMB.hpp
+//! \brief A hybrid version of the framework proposed by Gebremedhin, Manne, 
+//!        and Boman
+
+namespace Zoltan2 {
+
+template <typename Adapter>
+class AlgHybridGMB : public Algorithm<Adapter>
+{
+  public:
+  
+    typedef typename Adapter::lno_t lno_t;
```

Comment:
We have C++11 now; use `using`, not `typedef`.

Reply:
Switched all instances of typedef to using.